site/public/js/forum.js

```diff
@@ -774,6 +774,7 @@ function modifyOrSplitPost(e) {
     e.preventDefault();
     const form = $(this);
     const formData = new FormData(form[0]);
+    formData.append('deleted_attachments', JSON.stringify((getDeletedAttachments())));
```

Comment:
It looks like you have some redundant parentheses here:
```suggestion
    formData.append('deleted_attachments', JSON.stringify(getDeletedAttachments()));
```